python/tests/test_topology.py

```diff
@@ -49,6 +49,530 @@ def generate_segments(n, sequence_length=100, seed=None):
     return segs
 
 
+def kc_distance_tree(tree1, tree2, topo_v_age=0):
+    """
+    Returns the Kendall-Colijn distance between the specified pair of trees.
+    topo_v_age determines weight of topology vs branch lengths in calculating
+    the distance. See Kendall & Colijn (2016):
```

Comment:
Might be worth saying that 0 = all topology, 1 = all age

Reply:
good point, changed.